inductiva/inductiva
# inductiva logs supports --stdout and --stderr

This PR introduces the ability to filter log outputs with the following commands:

- `inductiva logs :id: --stdout`: Displays only the standard output (stdout) from the logs.
- `inductiva logs :id: --stderr`: Displays only the standard error (stderr) from the logs.
- `inductiva logs :id:`: Displays both stdout and stderr from the logs, with stderr messages highlighted in red.

### How to Test

1. **OpenFOAM Simulation:**
   - Run the OpenFOAM simulation as described in the [documentation](https://docs.inductiva.ai/en/latest/simulators/OpenFOAM.html).
   - Use the `--stdout` option to see the logs normally.
   - Use the `--stderr` option to verify that no errors appear (assuming there are no errors in the simulation).

2. **FDS Simulation:**
   - Run the FDS simulation as outlined in the [documentation](https://docs.inductiva.ai/en/latest/simulators/FDS.html).
   - Note that this simulator writes all output to stderr.
     - Running `inductiva logs :id:` will display all text in red since it comes from stderr.
     - Using `--stdout` will result in no output.
     - Using `--stderr` will display the logs normally.
